src/remotestorage.ts

```diff
@@ -49,12 +49,12 @@ function emitUnauthorized(r) {
 }
 
 /**
-* Check if interval is valid: numeric and between 1000ms and 3600000ms
+* Check if interval is valid: numeric and between 1s and 1hr inclusive
```

Comment:
I think this was intentionally more than 1 second and less than 1 hour actually. If it should be inclusive, then I think we should specify a more reasonable minimum bound here.

Reply:
1 second *is* probably faster than anyone needs - but 1001ms isn't much better. 

My thought is that the rate is best limited by the server - one could be syncing to a private server on a private network, and be free to use a lot of resources if it made sense.  A public server should have rate-limiting, to avoid misuse and defend against DOS attacks.